### ISSUE TITLE
chore:Incomplete Aria Elements

### ISSUE THREAD

**Issue by sombo20** [netojose/react-modal#37]
**ARIA attributes (aria-labelledby and aria-describedby) are not being used properly. They should be filled with meaningful values to improve accessibility.**
